content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -123,43 +120,44 @@ You can configure inter-cluster communication based on the
 example, if two clusters reside on the same underlying network, you can enable
 cross-cluster communication by simply configuring firewall rules.
 
-### DNS with Multiple Clusters
+### DNS with multiple clusters
 
 When a client application makes a request to some host, it must first perform a
 DNS lookup for the hostname to obtain an IP address before it can proceed with
 the request.
-
 In Kubernetes, the DNS server residing within the cluster typically handles
-this DNS lookup. Kubernetes `Service` configures the DNS server.
+this DNS lookup, based on the configured `Service` definitions.
 
-Istio ignores the IP address returned from the DNS lookup, however. Instead, it
-uses the list of active endpoints for the host and load balances across them.
+Istio uses the virtual IP returned by the DNS lookup to load balance
+accross the list of active endpoints for the requested service, taking into account any
+Istio configured routing rules.
 Istio uses either Kubernetes `Service`/`Endpoint` or Istio `ServiceEntry` to
-configure its internal mapping of hostname to IP addresses.
+configure its internal mapping of hostname to workload IP addresses.
 
 This two-tiered naming system becomes more complicated when you have multiple
 clusters. Istio is inherently multicluster-aware, but Kubernetes is not
 (today). Because of this, the client cluster must have a DNS entry for the
 service in order for the DNS lookup to succeed, and a request to be
 successfully sent. This is true even if there are no instances of that
-service's Pods running in the client cluster.
+service's pods running in the client cluster.
 
 To ensure that DNS lookup succeeds, you must deploy a Kubernetes `Service` to
 each cluster that consumes that service. This ensures that regardless of
 where the request originates, it will pass DNS lookup and be handed to Istio
 for proper routing.
-
 This can also be achieved with Istio `ServiceEntry`, rather than Kubernetes
-`Service`. `ServiceEntry` does not configure DNS, however. This means that
-DNS will have to be configured either manually or with automated tooling,
-such as the [Istio CoreDNS Plugin](https://github.com/istio-ecosystem/istio-coredns-plugin).
+`Service`. However, a `ServiceEntry` does not configure the Kubernetes DNS server
```

Comment:
I'm not sure that the new wording in this paragraph is an improvement. The sentence seems like a bit long-winded.

Reply:
I guess that's true, I split it into two sentences again.